utils/v2tofhirr4/mappings.bal

```diff
@@ -60,68 +60,108 @@ public function v2ToFhir(string|hl7:Message hl7) returns json|error {
 # + return - FHIR Bundle
 public function segmentToFhir(string segmentName, hl7:Segment segment) returns r4:BundleEntry[] {
     r4:BundleEntry[] entries = [];
+    V2toFhirMapperImpl impl = getMapperContext().getImpl();
     match segmentName {
         "NK1" => {
-            r4:BundleEntry entry = {'resource: nk1ToPatient(<Nk1>segment)};
-            entries.push(entry);
-            return entries;
+            Nk1ToPatient? nk1ToPatient = impl.nk1ToPatient;
+            if nk1ToPatient is Nk1ToPatient {
+                r4:BundleEntry entry = {'resource: nk1ToPatient(<Nk1>segment)};
+                entries.push(entry);
+                return entries;
```

Comment:
Since this is repeated in every block we can move these out of the `match` statement.

Reply:
Fixed with 4e64c6bbe86866f77e8b9c6c727389888e77d90d